packages/components/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - Added a new `ResizableBox` component.
 
+### Deprecation
+
+- `wp.components.PanelColor` has been deprecated in favor of `wp.editor.PanelColorSettings`.
```

Comment:
We should also add the deprecation information to /docs/reference/deprecated.md.

Reply:
Done!